src/ModAlgAss/MeatAxe.jl

```diff
@@ -234,8 +234,9 @@ end
 
 #  Restriction of the action to the submodule generated by C and the quotient
 function __split(C::T, G::Vector{T}) where T <: MatElem
-  action_sub = Vector{T}(undef, length(G))
-  action_quo = Vector{T}(undef, length(G))
+  View = only(Base.return_types(view, (T, UnitRange{Int}, UnitRange{Int})))
```

Comment:
Hm, this is a pattern we try to avoid, since `return_types` is not documented and part of any official interface. So it could vanish in any upcoming version or could start returning just `Any`.

One either has to introduce a helper function computing the type or refactor the loop below.